en/web.config

```diff
@@ -374,18 +374,18 @@
                     <conditions logicalGrouping="MatchAll" trackAllCaptures="false" />
                     <action type="Redirect" url="{R:1}/hierarchicalgrid/display-density" redirectType="Permanent" />
                 </rule>
-                <rule name="Redirect for Ignite UI Angular grid/excel_style_filtering" enabled="true" stopProcessing="true">
-                    <match url="^(.*components)/grid/excel_style_filtering.html$" />
+                <rule name="Redirect for Ignite UI Angular grid/excel-style-filtering" enabled="true" stopProcessing="true">
+                    <match url="^(.*components)/grid/excel-style-filtering.html$" />
                     <conditions logicalGrouping="MatchAll" trackAllCaptures="false" />
                     <action type="Redirect" url="{R:1}/grid/excel-style-filtering" redirectType="Permanent" />
                 </rule>
-                <rule name="Redirect for Ignite UI Angular treegrid/excel_style_filtering" enabled="true" stopProcessing="true">
-                    <match url="^(.*components)/treegrid/excel_style_filtering.html$" />
+                <rule name="Redirect for Ignite UI Angular treegrid/excel-style-filtering" enabled="true" stopProcessing="true">
+                    <match url="^(.*components)/treegrid/excel-style-filtering.html$" />
                     <conditions logicalGrouping="MatchAll" trackAllCaptures="false" />
                     <action type="Redirect" url="{R:1}/treegrid/excel-style-filtering" redirectType="Permanent" />
                 </rule>
-                <rule name="Redirect for Ignite UI Angular hierarchicalgrid/excel_style_filtering" enabled="true" stopProcessing="true">
-                    <match url="^(.*components)/hierarchicalgrid/excel_style_filtering.html$" />
+                <rule name="Redirect for Ignite UI Angular hierarchicalgrid/excel-style-filtering" enabled="true" stopProcessing="true">
+                    <match url="^(.*components)/hierarchicalgrid/excel-style-filtering.html$" />
                     <conditions logicalGrouping="MatchAll" trackAllCaptures="false" />
                     <action type="Redirect" url="{R:1}/hierarchicalgrid/excel-style-filtering" redirectType="Permanent" />
                 </rule>
```

Comment:
@zdrawku do we even need these redirects since this PR is adding "-" in names of excel topics?

Reply:
For the time being these redirects are needed. There is a process of backlink tracking that will rename most of the old URLs, BUT we need to have insurance if anyone, anywhere is still trying to reach the old URLs.

@mddifilippo89 I am changing back some of the redirects as I believe they were changes by mistake
https://github.com/IgniteUI/igniteui-docfx/pull/2399/commits/4ec8ead6e35b6edeaf17cdf34bf483bd656b4fe8